frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1699,4 +1699,4 @@ type GraphProps = {
 type MonitoringResourceIconProps = {
   className?: string;
   resource: MonitoringResource;
-};
+};
```

Comment:
Unintentional change? (I believe this causes the linter to fail.)

Reply:
When the codes were copied&pasted from  #7386, the newline was missed at the end of file.
After add the newline  at the end of file, it will pass prettier/prettier.

Reply:
Done